modules/installation-user-infra-machines-pxe.adoc

```diff
@@ -139,6 +139,11 @@ file on your TFTP server. The `coreos.inst.image_url` parameter value is the
 location of the compressed metal RAW image on your HTTP server, and the
 `coreos.inst.ignition_url` parameter value is the location of the bootstrap
 Ignition config file on your HTTP server.
++
+[NOTE]
+====
+To access serial console, append the correct `console=` setting for your hardware platform. Most users can use `console=ttyS0`.
```

Comment:
It'd be good to spell this out in a bit more detail.  Perhaps something like this:

```suggestion
This configuration does not enable serial console access on machines with a graphical console.  To configure a different console, add one or more `console=` arguments to the `APPEND` line.  For example, add `console=tty0 console=ttyS0` to set the first PC serial port as the primary console and the graphical console as a secondary console.  For more information, see https://access.redhat.com/articles/7212[this article].
```